collections/dominode-resources/checklists/legacy-raster.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "DomiNode - legacy raster validation checklist",
+  "description": "A checklist for validating legacy raster datasets",
+  "dataset_type": "raster",
+  "validation_artifact_type": "dataset",
+  "checks": [
+    {
+      "name": "legacy",
+      "description": "Check for legacy raster datasets.",
+      "guide": "This check will always defaults to True, it has been added for the validation of DomiNode legacy raster datasests",
+      "automation": {
+        "algorithm_id": "script:legacy",
```

Comment:
You need to update this to reflect the new name, which seems to be `script:noopvalidator` (please test it out)